src/main/java/com/iexec/core/replicate/Replicate.java

```diff
@@ -19,28 +19,30 @@
 import com.fasterxml.jackson.annotation.JsonIgnore;
 import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
 import com.iexec.common.chain.ChainReceipt;
-import com.iexec.common.replicate.ReplicateStatus;
-import com.iexec.common.replicate.ReplicateStatusCause;
-import com.iexec.common.replicate.ReplicateStatusDetails;
-import com.iexec.common.replicate.ReplicateStatusModifier;
-import com.iexec.common.replicate.ReplicateStatusUpdate;
-
+import com.iexec.common.replicate.*;
 import lombok.Data;
 import lombok.NoArgsConstructor;
 
-import java.util.*;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Date;
+import java.util.List;
 import java.util.stream.Collectors;
 
-import static com.iexec.common.replicate.ReplicateStatus.*;
-import static com.iexec.common.replicate.ReplicateStatusUpdate.*;
+import static com.iexec.common.replicate.ReplicateStatus.CREATED;
+import static com.iexec.common.replicate.ReplicateStatus.WORKER_LOST;
+import static com.iexec.common.replicate.ReplicateStatusUpdate.poolManagerRequest;
 
 
 @Data
 @NoArgsConstructor
 @JsonIgnoreProperties(ignoreUnknown = true)
 public class Replicate {
 
-    private List<ReplicateStatusUpdate> statusUpdateList;
+    private List<ReplicateStatusUpdate> statusUpdateList = new ArrayList<>(
```

Comment:
Add `FIXME` to pass this field `final` for immutability
To be done in another PR, breaks many tests in detector tests, `TaskUpdateManagerTest`, `ReplicateTests` and `ReplicateSupplyServiceTests`.
More analysis is required to replace calls to `setStatusUpdateList()`.

Reply:
Added there! https://github.com/iExecBlockchainComputing/iexec-core/pull/524/commits/dbcab6ac8f5b20da31a08674bd523f3c65bf1409